webpages/settings/components/addon-body.js

```diff
@@ -124,6 +177,13 @@ export default async function ({ template }) {
         this.$root.openRelatedAddons(this.addon);
         this.$root.blinkAddon(clickedAddon._addonId);
       },
+      toggleDropdown() {
+        this.isDropdownOpen = !this.isDropdownOpen;
+        this.$root.closePickers({ isTrusted: true }, null, {
+          callCloseDropdowns: false,
+        });
+        this.$root.closeResetDropdowns({ isTrusted: true }, this); // close other dropdowns
```

Comment:
This method should probably be renamed to `closeDropdowns` now that there's another kind of dropdown.